examples/encoding/infrastructure/create_simple_encoding_kubernetes.py

```diff
@@ -34,6 +35,7 @@ def main():
     s3_output = S3Output(access_key=S3_OUTPUT_ACCESSKEY,
                          secret_key=S3_OUTPUT_SECRETKEY,
                          bucket_name=S3_OUTPUT_BUCKETNAME,
+                         cloud_region=S3_CLOUD_REGION,
```

Comment:
Not required (default of `NONE` is working with the latest Python client)

Reply:
The default (`None`) is setting us-west-1. Although it's working, one may be confused if the object is in another region. What do you think?